docs/descriptors.md

```diff
@@ -13,9 +13,7 @@ Schema descriptors are defined in XDM using the `SchemaDescriptor` schema.
 
 While schema descriptors can be used to define metadata about a single schema, they are also common used to describe relationships between schemas. This mechanism can be used to link schemas together at the property level, defining the equivalent of "foreign key" relationships in a relational database.
 
-The following relationship types are defined by XDM:
-
-TBD
+Details on relationship descriptors and their usage can be found [here](../schemas/common/descriptors/relationshipdescriptor.description.md).
```

Comment:
I'd point this to the generated docs, i.e. `./reference/common/descriptors/relationshipdescriptor.schema.md